### ISSUE TITLE
Detect E0364/E0365 errors

### ISSUE THREAD

**Issue by Undin** [intellij-rust/intellij-rust#5759]
## Environment

* **IntelliJ Rust plugin version:** 0.3.127.3225-202-nightly
* **Rust toolchain version:** 1.45.0 (5c1f21c3b 2020-07-13) x86_64-apple-darwin
* **IDE name and version:** IntelliJ IDEA 2020.2 Beta Ultimate Edition (IU-202.6397.20)
* **Operating system:** macOS 10.13.6

## Problem description
Currently, [E0364](https://doc.rust-lang.org/error-index.html#E0364) and [E0365](https://doc.rust-lang.org/error-index.html#E0365) are not detected by the plugin as compilation errors. It would be great to highlight cases when a reexported item is not public and provide quick fix to add `pub` modified

## Steps to reproduce
```rust
mod foo {
    mod bar {}
    pub use bar as baz;  // <--- error[E0365]: `bar` is private, and cannot be re-exported
    struct S;
    pub use S as T; // <--- error[E0364]: `S` is private, and cannot be re-exported
}
```
